docs/snippets/tracer/basicUsage.ts

```diff
@@ -3,5 +3,5 @@ import { Tracer } from '@aws-lambda-powertools/tracer';
 const tracer = new Tracer({ serviceName: 'serverlessAirline' });
 
 export const handler = async (_event, _context): Promise<void> => {
-    // ...
+  // ...
```

Comment:
Let's remove this empty comment (`// ...`) and do something with the tracer i.e. `tracer.getSegment();`